LayoutTests/platform/wincairo/TestExpectations

```diff
@@ -896,8 +896,6 @@ http/tests/xmlhttprequest/cross-origin-authorization.html [ Failure ]
 http/tests/xmlhttprequest/cross-origin-cookie-storage.html [ Failure ]
 http/tests/xmlhttprequest/cross-origin-no-authorization.html [ Failure ]
 http/tests/xmlhttprequest/logout.html [ Failure ]
-http/tests/xmlhttprequest/origin-header-same-origin-get-async.html [ Failure ]
-http/tests/xmlhttprequest/origin-header-same-origin-get-sync.html [ Failure ]
```

Comment:
This only fixes two tests (which it looks like both depend on access-control-basic-allow-access-control-origin-header.cgi)? 🤔 Are problems observed with the rest of the test files here?

Reply:
I don't know how many tests pass by this change. WinCairo still skips a lot of tests under http/tests. The one of the reasons is this text mode problem. I have to check other test results after fixing Perl and Python CGI problems.